onnxscript/test/function_libs/torch_aten/ops_correctness_test.py

```diff
@@ -171,6 +171,18 @@ def _amax_amin_input_wrangler(
     return args, kwargs
 
 
+def _arange_input_wrangler(
+    args: list[Any], kwargs: dict[str, Any]
+) -> tuple[list[Any], dict[str, Any]]:
+    new_args = []
+    for arg in args:
+        if isinstance(arg, int):
+            # Explicitly convert to int64 because ints are 32-bit on Windows
```

Comment:
nit:

...because int type is 32-bit on Windows.

Reply:
Done